src/3p-frame-messaging.js

```diff
@@ -51,6 +51,9 @@ export const MessageType = {
   // For amp-analytics' iframe-transport
   SEND_IFRAME_TRANSPORT_EVENTS: 'send-iframe-transport-events',
   IFRAME_TRANSPORT_EVENTS: 'iframe-transport-events',
+
+  // For user-error
+  USER_ERROR: 'user-error',
```

Comment:
Can we rename this to `USER_ERROR_IN_IFRAME`? I find that to be more clear. Thanks